### DISCUSSION TITLE
How to add a synchronous helper function?

### DISCUSSION THREAD

**Discussion by marinakuranda** [cypress-io/cypress#9426] [Questions and Help]
Hello!

I want to use some shared logic across my tests. 

But the way it is done is redundant to use commands.

Because commands are asynchronous and to rewrite a command to work in a synchronous seems redundant.

This is what I've added to my commands file:

```js
Cypress.Helpers = {};

Cypress.Helpers.parsePrice = function parsePrice(str) {
  return parseFloat(
    // 'some logic
  );
}
```

It seems to work the expected way, but it is undocumented.

Do you recommend another approach for shared synchronous functions?

**Comment by bahmutov** [comment]
What you are doing here (attaching a method to the global Cypress object) works, but I would advise to simply have common functions and require or export them just like plain JavaScript functions. 

```js
// cypress/support/utils.js
export const parsePrice = (str) => parseFloat(/* some logic */)

// cypress/integration/specA.js
import {parsePrice} from '../support/utils'
// use parsePrice in your tests
```
